lib/Graph/Graph.cpp

```diff
@@ -423,14 +425,30 @@ static void assertConvDims(NodeValue input, NodeValue filter, NodeValue bias,
   assert(bias->getType()->size() == filterDims[0] && "Invalid bias size");
 }
 
+ConvolutionNode *
+Function::createConv(llvm::StringRef name, NodeValue input, NodeValue filter,
```

Comment:
do we really need to expose that many API for creating convolution node?

Reply:
According to @jfix71 's suggestion,  since there are many equal-padding cases, it is better to provide the signal padding one. I think this make sense. Since in Caffe2, there are param "pad" for single padding, and if it is non-square padding, it has pad_t, pad_l, pad_b,pad_r  4 params.